recipes/ghc/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ghc" %}
+{% set version = "8.2.2" %}
+{% set sha256 = "bb8ec3634aa132d09faa270bbd604b82dfa61f04855655af6f9d14a9eedc05fc" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://https://downloads.haskell.org/~ghc/{{ version }}/{{ name }}-{{ version }}-src.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - toolchain
+    - gmp
+  run:
+    - gmp >=5.0.1,<7
+
+test:
+  commands:
+    - ghc --help
+
+about:
+  home: https://haskell.org/ghc/
+  license: The Glasgow Haskell Compiler License
+  summary: 'Glorious Glasgow Haskell Compilation System'
+  doc_url: https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/
+  dev_url: https://github.com/ghc/ghc
```

Comment:
You could also add `license_file: LICENSE`

Reply:
ah, thanks for the quick review :-) I will add your suggestions